var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -69,6 +69,9 @@ class Perl(Package):  # Perl doesn't use Autotools, it should subclass Package
     variant('cpanm', default=True,
             description='Optionally install cpanm with the core packages.')
 
+    variant('useshrplib', default=False,
```

Comment:
I spent about 30 seconds trying to parse `useshrplib` in my mind and started off with `uses hrp lib` before realizing it was "use shared perl lib". I would argue towards `shared`.